Yasgui/packages/utils/src/index.ts

```diff
@@ -73,6 +73,22 @@ export class TranslationService {
   }
 }
 
+export class TimeFormattingService {
+  private static _INSTANCE: TimeFormattingService;
+
+  static get INSTANCE(): TimeFormattingService {
+    if (!TimeFormattingService._INSTANCE) {
+      TimeFormattingService._INSTANCE = new TimeFormattingService();
+    }
+    return TimeFormattingService._INSTANCE;
+  }
+
+  getHumanReadableSeconds(millisecondsAgo: number, preciseSeconds = false): string {
+    return "";
```

Comment:
These should have a more meaningful default. You can just return the argumanent as a string or something.

Or throw an exception if it must be overiden.

Reply:
Done